modules/KalturaSupport/components/dualScreen/dualScreen.js

```diff
@@ -105,7 +105,11 @@
                             }, 500);
                         }
                     }
-				} );
+                    if( _this.getPlayer().isMobileSkin() ) {
```

Comment:
@shalommeoded 
I do not want to change the controllbar on any player that has this plugin (this is what your code will do) 
I rather do this with targeting the 'dualScreenControlBar' class and not controlBar.
Besides that - whats the point doing this in JQ and not as a regular well-targeted css? We have a .mwPlayerContainer.mobile.mobileSkin that we can use as a high-level umbrella.

Reply:
@eitanavgil made the changes, please review.